### PR TITLE
test: surface capped source stroke widths

### PR DESCRIPTION
**Pull request by ebelo**
## Summary

- Add raw/capped source line-width metadata to the path/pedestrian focus report when sampled Mapbox source widths hit qfit's 3 mm QGIS cap.
- Keep the existing capped `line-width` deltas unchanged while making the z18 pedestrian core/case cap relationship explicit before any production styling change.
- Update regression coverage for the sampled controls, Markdown output, and the minimum-width floor edge case flagged by Greptile.

## Validation

- `python3 -m pytest tests/test_mapbox_outdoors_path_pedestrian_focus.py -q --tb=short`
- `python3 -m pytest tests/ -x -q --tb=short`
- `git diff --check`
- `python3 scripts/package_plugin.py`
- `python3 validation/mapbox_outdoors_path_pedestrian_focus.py --road-features-json debug/mapbox-outdoors-road-features/all-cameras/20260520T202841Z/road-features.json --source-style-json debug/mapbox-outdoors-source-style/20260520T042926Z/mapbox-outdoors-v12.json --qgis-style-json zermatt-trails-z18-outdoors=debug/mapbox-outdoors-comparison-steps-z18-width/zermatt-trails-z18-outdoors/20260521T014450Z/qgis-preprocessed-style.json --qgis-label-styles-json zermatt-trails-z18-outdoors=debug/mapbox-outdoors-comparison-steps-z18-width/zermatt-trails-z18-outdoors/20260521T014450Z/qgis-label-styles.json`

## Evidence

- Refreshed report: `debug/mapbox-outdoors-path-pedestrian-focus/20260521T020350Z/summary.md`
- The z18 `road-pedestrian` row now shows source `line-width_raw_mm=3.175` capped to `line-width=3.0`.
- The z18 `road-pedestrian-case` row now shows source `line-width_raw_mm=3.8364583333333333` capped to `line-width=3.0`.

### CODE REVIEW

File: validation/mapbox_outdoors_path_pedestrian_focus.py [hunk 1]
```diff
@@ -1055,6 +1056,19 @@ def _source_sampled_stroke_controls(
...
+def _source_line_width_controls(line_width_expr: object, camera_zoom: float) -> dict[str, object]:
+    line_width_px = _extract_zoom_scalar_size_at_zoom(line_width_expr, camera_zoom)
+    if line_width_px is None:
+        return {}
+    raw_width_mm = line_width_px * _MAPBOX_PIXEL_TO_MM
+    capped_width_mm = max(0.1, min(raw_width_mm, _MAX_LINE_WIDTH_MM))
+    controls: dict[str, object] = {"line-width": capped_width_mm}
+    if abs(capped_width_mm - raw_width_mm) > 1e-12:
+        controls["line-width_raw_mm"] = raw_width_mm
+        controls["line-width_capped"] = True
```
[the remark this answers is not in the document]
Comment (ebelo) [reply validation/mapbox_outdoors_path_pedestrian_focus.py]: Fixed in 79606fa by narrowing `line-width_capped` to the 3 mm ceiling case and adding a regression test for the 0.1 mm floor path.
